server/list.go

```diff
@@ -0,0 +1,239 @@
+package main
+
+import (
+	"fmt"
+)
+
+const (
+	// MyListKey is the key used to store the order of the owned todos
+	MyListKey = ""
+	// InListKey is the key used to store the order of received todos
+	InListKey = "_in"
+	// OutListKey is the key used to store the order of sent todos
+	OutListKey = "_out"
+)
+
+// ListStore represents the KVStore operations for lists
+type ListStore interface {
+	AddItem(item *Item) error
+	GetItem(itemID string) (*Item, error)
+	RemoveItem(itemID string) error
+
+	GetItemOrder(userID string, itemID string, listID string) (*OrderElement, int, error)
```

Comment:
```suggestion
	GetItemOrder(userID, itemID, listID string) (*OrderElement, int, error)
```
You can simplify the parameter list like this.

Reply:
0/5 I like it better when every argument has its type near. Nevertheless, I can change it.

Reply:
It's more idiomatic to try to reduce the amount of text to read. You get used to it. :)